pennylane/devices/default_clifford.py

```diff
@@ -254,9 +309,10 @@ def __init__(
         self._check_clifford = check_clifford
 
         self._tableau = tableau
+        self._prob_states = None
 
-        seed = np.random.randint(0, high=10000000) if seed == "global" else seed
-        self._rng = np.random.default_rng(seed)
+        self._seed = np.random.randint(0, high=10000000) if seed == "global" else seed
```

Comment:
Why a seed and an rng?

Reply:
We use stim's `sampler` method for generating samples in the finite shot case, which accepts `seed` and not an `rng`. About the `rng`, I was using it for generating `seed` in the multiprocessing case.

Reply:
If we are feeding the same seed to the sampler each time, will the samples be identical each time we execute?

Reply:
``` py
@qml.qnode(qml.device('default.clifford', shots=50, seed=123455))
def circuit():
    qml.Hadamard(0)
    qml.S(0)
    return qml.expval(qml.PauliZ(0))

circuit(), circuit(), circuit()
```
Tried to fix this. It should be fine now. Above circuit would now give - 

``` py
(tensor(-0.28, requires_grad=True),
 tensor(-0.12, requires_grad=True),
 tensor(0., requires_grad=True))
```